torch/csrc/distributed/c10d/init.cpp

```diff
@@ -249,6 +249,15 @@ to apply layer-wise operations.
 Returns:
     Whether this bucket is the last bucket to allreduce in an iteration.
     This also means that this bucket corresponds to the first few layers in the forward pass.
+)")
+      .def(
+          "set_tensor",
+          &::c10d::GradBucket::setTensor,
+          py::arg("tensor"),
+          py::arg("i"),
+          py::call_guard<py::gil_scoped_release>(),
+          R"(
+Replaces _tensor at index i with tensor input.
```

Comment:
There is only "tensors_" member, no "_tensor" member in `GradBucket` class.

Nit: Replaces the ith tensor in the bucket with the input tensor.